src/vs/workbench/contrib/inlineChat/browser/inlineChatDecorations.ts

```diff
@@ -90,20 +93,25 @@ export class InlineChatDecorationsContribution extends Disposable implements IEd
 		this._gutterDecorationOpaque.glyphMarginHoverMessage = hoverMessage;
 	}
 
-	private _onEnablementOrModelChanged(): void {
+	private _updateCurrentBreakpoints(uri: URI) {
+		this._currentBreakpoints = [...this._debugService.getModel().getBreakpoints({ uri })];
+	}
+
+	private _onEnablementOrModelChanged(executeOnlyDisposal: boolean = false): void {
```

Comment:
`_onEnablementOrModelChanged` can be called any time, by other listeners as well. At that time it needs to know if the inline chat session is on or off.
- > create a property 'hasInlineChatSession` that is maintained by `onWillStartSession` and `onOnDidSession`

Reply:
That's right I just noticed, I'll add that.

Reply:
Actually there is the method getSession(editor: ICodeEditor, uri: URI) which I can use